src/ui/common/form/RenderRadioInput.js

```diff
@@ -27,10 +27,11 @@ const buttonToolbar = (input, toggleElement, defaultValue) => (
 const RenderRadioInput = ({
   input, append, label, labelSize, alignClass,
   meta: { touched, error }, help, toggleElement, defaultValue,
+  'data-testid': dataTestId,
```

Comment:
**Never** pass `data-testid` as a prop. It must be totally separated from Component interface